### ISSUE TITLE
West Nile virus antibody positive

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10288]
**Preferred term label:**
West Nile virus antibody positive

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://uts.nlm.nih.gov/uts/umls/concept/C1328571

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
West Nile

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

added as
Anti-West Nile virus antibody positivity
HP:0430053
